package-template/README.md

```diff
@@ -0,0 +1,27 @@
+This folder contains details about the template that should be used for each newly forked Pulsar Package repository.
+
+The files in this folder should be used as a loose to strict structure of new files to add to any given repo, with these files intending to accomplish the following:
+
+* Give proper attribution to the original maintainer of the forked package
+* Specify the level of support, and the expected method of interaction with the repository
+* The GitHub Action workflows to add to the forked repo to ensure a hands off approach is possible.
+
+With our goals stated, lets get to it, with the first template provided being below:
+
+---
+
+<!-- Pulsar Cooperative Package Repository Template, place at the top of the orignal README -->
+
+# Pulsar Cooperative
```

Comment:
Should this be `Pulsar Cooperative` or just `<Package Name Here>`?.

Reply:
Hmm. I'd say `Pulsar Cooperative`, since the name of the package will likely already be present in the readme as a header, meanwhile this section is detailing the org itself, then can quickly move on to the actual package content.

Reply:
It seems a bit odd to me to have an `h1` as something that isn't the name of the package is all. Not sure how all the various markdown renderers play ball but some dont like having multiple `h1`s.

Reply:
I went with your suggestion of using the `NOTE` item, and removing the extra `h1` so we should be good here